web/src/frontend/scripts/mock_data/miljovariabler.json

```diff
@@ -12,5 +12,7 @@
 	"soknadinnsending.soknad.path": "http://127.0.0.1:8282/soknadinnsending/soknad",
 	"dialogarena.cms.url": "https://appres-q1.nav.no",
 	"dittnav.link.url": "http://www.nav.no/dittnav",
-	"saksoversikt.link.url": "http://127.0.0.1:8443/saksoversikt"
+	"saksoversikt.link.url": "http://127.0.0.1:8443/saksoversikt",
+	"feature.toggle.vise_tps_kontonummer": true,
+	"feature.toggle.beOmLonnslippVedlegg": true
```

Comment:
blander mellom camelCase og snake_case.
Konvensjonen så langt er å lowercase, og .enabled til slutt for å vise at man spesifikt må skru den på, f. eks
feature.tpskontonummer.enabled = true